### ISSUE TITLE
About script_generative_modifier.sh 

### ISSUE THREAD

**Issue by minjung98** [naver/posescript#15]
Hello. Thank you for your clear and prompt responses as always. Your help has greatly benefited my research. 
This issue pertains to an error in the generative_modifier/script_generative_modifier.sh file.

`generative_modifier/script_generative_modifier.sh: line 74: 128: command not found
generative_modifier/script_generative_modifier.sh: line 79: syntax error near unexpected token 'fi'
generative_modifier/script_generative_modifier.sh: line 79: `

These are the error messages I'm encountering. Have you ever faced such errors before? It seems the error is related to the fi part, but as far as I can tell, all if statements are correctly closed with fi. I'm not sure how to resolve this error.

The command I used to execute the script was:
`bash generative_modifier/script_generative_modifier.sh 'train' pretrain 1
`
Thank you.

**Comment by g-delmas** [comment]
Here is a shot in the dark: can you try with `sh generative_modifier/script_generative_modifier.sh 'train' pretrain 1` ? (Note that `bash` was replaced by `sh`)

**Comment by minjung98** [comment]
Maybe it does not work.. 
<img width="570" alt="스크린샷 2024-06-15 오전 1 10 30" src="https://github.com/naver/posescript/assets/130021582/067b74ad-6d19-4752-a977-891a88eee699">
I got error message like this. 

I have an additional question. For the model that generates motion feedback, I understand that the evaluation metrics mentioned in the paper are BLEU-4, ROUGE-L, and METEOR. According to Table 6 of the paper, the model achieved fairly high scores of approximately 12, 33, and 31, respectively. However, after training for 3000 epochs, the evaluation on the validation set is as follows:
`{
  "epoch": 2999,
  "time": 1718292528.915231,
  "lr": 0.0001,
  "train_lr": 0.00010000000000000145,
  "train_loss": 0.8373838120796642,
  "val_loss": 0.8025384304720328,
  "val_fake_loss": 14.238175077175875,
  "avg_likelihood": -0.642828769547392,
  "kld_a": 1.6087877553480452,
  "kld_b": -0.5283231214943266,
  "fid": 0.017062776868283835,
  "v2v_elbo": 2.2433030600488797,
  "v2v_dist_avg": 0.21313080319745648,
  "v2v_dist_top1": 0.12490824154582927,
  "v2v_dist_top6": 0.14983244631037013,
  "jts_elbo": 1.7913970198350309,
  "jts_dist_avg": 0.29169768930144785,
  "jts_dist_top1": 0.15103213335187873,
  "jts_dist_top6": 0.19051245852644813,
  "rot_elbo": 1.258873670046123,
  "rot_dist_avg": 9.254753686692796,
  "rot_dist_top1": 6.998022135010912,
  "rot_dist_top6": 7.6958002690953995,
  "ret_r1_prec": 89.4573307319893,
  "ret_r2_prec": 94.43601416287304,
  "ret_r3_prec": 96.59657489703012,
  "ret_multimodality": 0.40695830595373583,
  "bleu": 0.2857213691929439,
  "rougeL": 0.37234075656443766,
  "meteor": 0.39688325648956796,
  "bertscore": 0.9066700533304836
}
`
These scores are significantly lower compared to other examples. Could you please explain why the results are like this? For your information, I have not made any changes to your code; I am simply reproducing it to understand the code.

Thank you.

**Comment by g-delmas** [comment]
In issue 14, it seems like you were able to run the script. Did something change in between?

As for the dictionary of results, the scale of the BLEU, ROUGE and METEOR metrics seems okay. The numbers in the table were reported after [a bit of formatting](https://github.com/naver/posescript/blob/main/src/text2pose/evaluate.py#L51).   As a side note, numbers obtained from the validation set cannot be compared directly to that of the paper, since tables report numbers on the (bigger) test set -- you are probably aware of that, I'm writing it here just in case.

**Comment by minjung98** [comment]
Now I understood your explanation regarding the BLEU, ROUGE, and METEOR metrics.
In Issue 14, I was able to run the script using `bash generative_modifier/script_generative_modifier.sh 'train' pretrain 1`. However, as you mentioned, when I run the command `sh generative_modifier/script_generative_modifier.sh 'train' pretrain 1`, I encounter the syntax error I showed you in the screenshot.

Even when running `bash generative_modifier/script_generative_modifier.sh 'train' pretrain 1`, it ran well for 3000 epochs, but then it seems there was a problem with the fi statement afterward.
